src/Microsoft.ML.Data/DataLoadSave/TransformerChain.cs

```diff
@@ -252,8 +252,47 @@ public static TransformerChain<ITransformer> LoadFrom(IHostEnvironment env, Stre
         {
             using (var rep = RepositoryReader.Open(stream, env))
             {
-                ModelLoadContext.LoadModel<TransformerChain<ITransformer>, SignatureLoadModel>(env, out var transformerChain, rep, LoaderSignature);
-                return transformerChain;
+                try
+                {
+                    ModelLoadContext.LoadModel<TransformerChain<ITransformer>, SignatureLoadModel>(env, out var transformerChain, rep, LoaderSignature);
+                    return transformerChain;
+                }
+                catch
+                {
+                    var chain = ModelFileUtils.LoadPipeline(env, stream, new MultiFileSource(null), extractInnerPipe: false);
```

Comment:
Usually when you do something that has a bad code smell (e.g., some sort of global `try/catch` of literally any type of exception ***whatsoever***) it is either best to either (1) explain why it must be so or (2) try not to do it. :) Either is fine with me.